libs/schemas/src/utils.ts

```diff
@@ -38,10 +38,7 @@ export const linksSchema = {
 
 export const PG_INT = z.number().int().min(MIN_SCHEMA_INT).max(MAX_SCHEMA_INT);
 export const zBoolean = z.preprocess((v) => v && v !== 'false', z.boolean());
-export const zDate = z.preprocess(
-  (arg) => (typeof arg === 'string' ? new Date(arg) : arg),
-  z.date(),
-);
```

Comment:
There are a lot of instances where this is used. 

The newer `z.coerce.date();` doesn't give errors on `numbers` and `null` types -- but the removed validation does. Not sure what the effects would be, I would suggest we find out the effects or revert this.